config/config.exs

```diff
@@ -51,7 +51,7 @@ config :tailwind,
 # Configures Elixir's Logger
 config :logger, :console,
   format: "$time $metadata[$level] $message\n",
-  metadata: [:request_id, :project, :external_id, :application_name, :sub, :error_code]
+  metadata: [:request_id, :project, :external_id, :application_name, :sub, :exp, :iss, :error_code]
```

Comment:
Do you actually need this though? I think this is only necessary to always expose the metadata if set through `Logger.metadata` I think.

When it's sent explicitly logger will log any metadata I think. 

For example here we log "measurements" and it's not added here

https://github.com/supabase/realtime/blob/main/lib/realtime/telemetry/logger.ex#L33-L34

Reply:
you are right, this would only be needed for development where seeing this is not that relevant to be fair... removed it